pkg/cli/components/advisory/prompt/model.go

```diff
@@ -46,14 +46,8 @@ func (m Model) newPackageFieldConfig() field.TextFieldConfiguration {
 	}
 }
 
-var cveIDRegex = regexp.MustCompile(`^CVE-\d{4}-\d{4,}$`)
-
-// grabbed this regex from Github Docs
-// see https://docs.github.com/en/code-security/security-advisories/global-security-advisories/about-the-github-advisory-database#about-ghsa-ids
-var ghsaRegex = regexp.MustCompile(`GHSA(-[23456789cfghjmpqrvwx]{4}){3}`)
-
-var ValidCVEID = field.TextValidationRule(func(value string) error {
-	if !cveIDRegex.MatchString(value) && !ghsaRegex.MatchString(value) {
+var validCVEID = field.TextValidationRule(func(value string) error {
+	if !vuln.RegexCVE.MatchString(value) && !vuln.RegexGHSA.MatchString(value) {
```

Comment:
Should we use `vuln.ValidateID`, or do we specifically not want to allow GO entries

Reply:
Good question. We don't have good reason to disallow GO- entries, we just haven't used them yet. But we're already using alternative IDs (i.e. GHSA), so we might as well allow GO here, too. I'll update this to use `vuln.ValidateID`.

Reply:
oooo this ends up collapsing this stanza down to:

```go
var validCVEID = field.TextValidationRule(vuln.ValidateID)
```

🤩